compute/cker/include/cker/operation/reference/BatchMatMul.h

```diff
@@ -107,6 +138,60 @@ inline void BatchMatMul(const Shape &lhs_shape, const float *lhs_data, const Sha
   }
 }
 
+#if defined(CKER_X86_PLATFORM)
+inline void bmm_optimized(const BMMParams &bmm_params, const float *lhs_data, const float *rhs_data,
+                          float *output_data)
+{
+  MatrixParams<float> lhs_params;
+  lhs_params.order = Order::kRowMajor;
+  lhs_params.rows = bmm_params.lhs_rows;
+  lhs_params.cols = bmm_params.lhs_cols;
+
+  MatrixParams<float> rhs_params;
+  rhs_params.order = Order::kRowMajor;
```

Comment:
```suggestion
  rhs_params.order = Order::kColMajor;
```

@tomdol Could you please check this ? (AFAIU, batch matmul's second input is normally col major. Let me know if wrong)

Reply:
I have some results that I described here https://github.com/Samsung/ONE/pull/14370 but long story short - this field doesn't matter for Gemm with Eigen so I believe setting of the `order` field could be removed from this kernel entirely.